pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1364,6 +1364,9 @@
         "xamarinmac20": "Any",
         "xamarintvos10": "Any",
         "xamarinwatchos10": "Any"
+      },
+      "AssemblyVersionInPackageVersion": {
```

Comment:
Does this need 
```
      "InboxOn": {},
```
?
Like in 4e408ff42f9ecbd63f6787e781d6f06bb28ca81e

Reply:
There's already an InboxOn line in the PerformanceCounter section, so I thought it wasn't necessary.

Reply:
Ok, now I'm getting errors saying that we shouldn't be supporting PerformanceCounter on xamarin, etc, so I'll remove those lines in the InboxOn section.